scripts/check-region-info-compatibility.ts

```diff
@@ -21,6 +21,12 @@ function main(oldPackage: string, newPackage: string) {
   const disappearedFacts = oldFacts
     .filter((oldFact) => !newFacts.some((newFact) => factEq(oldFact, newFact)))
     .map((fact) => ({ fact, key: `${fact[0]}:${fact[1]}` }))
+    // These aren't accessed directly and we've just updated our handling of them,
+    // not removed this functionality. The mapping is unnecessary.
```

Comment:
Can you clarify which mapping this is referring to?

Reply:
We generate it at build time. I'll clarify.